handler.py

```diff
@@ -14,38 +16,120 @@
 SQS_CLIENT = boto3.client('sqs')
 
 
-def runScheduledObservatoryScan(event, context):
+def addPortScanToQueue(event, context):
+    data = json.loads(event['body'])
+    target = Target(data.get('target'))
+
+    if target.isValid():
+        print(SQS_CLIENT.send_message(
+              QueueUrl=os.getenv('SQS_URL'),
+              MessageBody="portscan|" + target.targetname
+              ))
+
+        return Response({
+            "statusCode": 200,
+            "body": json.dumps({'OK': 'target added to the queue'})
```

Comment:
I'm glad you aren't echoing input here and staying generic.

I am curious though whether we want consider offering up an identifier (or not) to users submitting a scan type (such as a uuid or something that effect, which might aid in troubleshooting/debugging if something doesn't go as planned)?

Reply:
Sounds like a good idea. Will add UUID to the response

Reply:
I have implemented this for the API endpoints, but not for scheduled scans. We can extend it to all job/scan types later if required.